src/main/java/com/sfuentes/Theater.java

```diff
@@ -0,0 +1,30 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+
+public class Theater {
+
+  ArrayList<Integer> ages;
+  final double price = 20d;
+
+  public Theater(ArrayList<Integer> age) {
```

Comment:
The name of the parameter should be in plural since it is a list

Reply:
fixe